infrastructure/scripts/create-testdata/create-release.sh

```diff
@@ -9,6 +9,9 @@ set -o pipefail
 
```

Comment:
regarding the PR description:
I assume the backend PR comes before the frontend PR. Therefore, I would remove the screenshot from the backend PR description.